config/main.py

```diff
@@ -3169,7 +3170,7 @@ def reload(ctx, no_dynamic_buffer, no_delay, dry_run, json_data, ports, verbose)
 
     log.log_info("'qos reload' executing...")
     if not dry_run:
-        _clear_qos(delay = not no_delay, verbose=verbose)
+        clear_qos_status = _clear_qos(delay = not no_delay, verbose=verbose)
```

Comment:
maybe just `status` suffice.
in the future perhaps we will have other steps that can fail.

Reply:
Agree, will use status directly.